forte/processors/machine_translation_processor.py

```diff
@@ -49,6 +49,8 @@ def __init__(self) -> None:
     # pylint: disable=unused-argument
     def initialize(self, resources: Optional[Resources],
```

Comment:
If this processor can only call MS translation, maybe it should be named more specifically. Or you can create an abstract MT processor that handle some basic operations.

Reply:
I have renamed it to `MicrosoftBingTranslator` in ebab198